deploy/SetupVmruntime.sh

```diff
@@ -151,7 +151,6 @@ startRuntime() {
                 DOCKER_RUN_CMD="docker run --security-opt apparmor=${APPARMOR_PROFILE_LIBVIRTD}"
         fi
 	${DOCKER_RUN_CMD} --net=host --privileged --pid=host --uts=host --ipc=host --user=root \
-	--security-opt apparmor=${APPARMOR_PROFILE_LIBVIRTD} \
```

Comment:
Why is this removed, is this related to the new version?

Reply:
this was actually a bug. the DOCKER_RUN_CMD has the proper setting for the --security-opt already, so this line is a dup-- and actually affect the deployment.